coast/TIDEGAUGE.py

```diff
@@ -308,19 +311,17 @@ def create_multiple_tidegauge(cls, file_list, date_start=None, date_end=None):
         tidegauge_list = []
         for file in file_to_read:
             try:
-                dataset = cls.read_gesla_to_xarray_v3(file, date_start, date_end)
-                new_object = TIDEGAUGE()
-                new_object.dataset = dataset
+                new_object = TIDEGAUGE(file, date_start, date_end, config)
                 tidegauge_list.append(new_object)
-            except:
-                # Problem with reading file: file TODO: add debug message here
-                pass
+            except FileNotFoundError as fne:
```

Comment:
We should avoid using bare excepts. These will catch **everything.** Including system exits.